### DISCUSSION TITLE
Property Editor not change options

### DISCUSSION THREAD

**Discussion by piovezanfernando** [eclipse-birt/birt#1019] [General]
Hi, 

Hi, I've been using Birt for a long time now and I was using version 4.6 until last year, but now I decided to test the new version 4.9 and 4.10, it's very good now because I have viewReport working on Linux which I didn't have in version 4.6.
I have a problem with the property editor, the first component that I select when opening the file is selected and the property editor options no longer change, for example if when opening the file I select a table, the properties of this table appear, but when selecting another component a text for example continues to display the properties of the table or point it is also if I select one of the "border" or "margin" these options are displayed after the options on the General tab. As you can see in the print, a text component is selected and the properties are from the table I selected when the file was opened.

![image](https://user-images.githubusercontent.com/38543704/181303665-ffb1c638-8824-4ff6-9101-fba8c4b81563.png)

This case is happening in the two tests I did the scenarios are

Linux Pop!_OS 22.04 LTS
GNOME 42
openjdk 11.0.15 2022-04-19
OpenJDK Runtime Environment (build 11.0.15+10-Ubuntu-0ubuntu0.22.04.1)
OpenJDK 64-Bit Server VM (build 11.0.15+10-Ubuntu-0ubuntu0.22.04.1, mixed mode, sharing)
Eclipse BIRT Project 4.9.0.v202203150031

And the second
Windows 7 64
Java 1.8
Eclipse BIRT Project 4.9.0.v202203150031

If anyone knows what may be happening to cause this error, I would be very grateful because I am unable to follow the use of this version

Thanks

**Comment by hvbtup** [comment]
Are you sure about your second test case? BIRT >= 4.9 needs Java >= 11 and will not work with Java 8...

**Reply by piovezanfernando** [reply]
Hi @hvbtup exactly on windows eclipse opened normally as java 1.8, on linux I don't know because I don't have this version.

**Comment by claesrosell** [comment]
This seems like a big problem, however, I cannot reproduce it in my environment. I am using Fedora 36 with Gnome 42.
How do you access Birt? Is it the "Birt report designer", the "Birt report designer all in one", or do you install Birt inside an existing Eclipse?
Can you reproduce this with any report file, or is it just this one file that causes this problem? If so: can you share it with us?

**Reply by piovezanfernando** [reply]
Hi @claesrosell I managed to solve the problem, I started a new project and copied only the report files to this new project and everything worked normally, I believe that some configuration that was in the .metadata folder of the previous project was causing this problem with eclipse. The previous project had also been created with the same version of Birt Report Design that I created the second one, possibly it was some interface color configuration that I had changed that caused this problem, so I decided not to mess with the interface colors even though it was fine bad to view

thanks for the help
